Microsoft.Toolkit.Uwp/Helpers/EnumHelper.cs

```diff
@@ -0,0 +1,47 @@
+﻿// ******************************************************************
```

Comment:
I do not see the value added of this extension. 
```
var values = Enum.GetValues(typeof(TEnum)).Cast<TEnum>().ToList();
```
The code below is already functional. I think you can remove this class